File: app/src/main/java/com/tebet/mojual/view/history/HistoryFragmentProvider.java
package com.tebet.mojual.view.history;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class HistoryFragmentProvider {

    @ContributesAndroidInjector(modules = HistoryFragmentModule.class)
    abstract HistoryFragment provideHistoryFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/history/HistoryFragmentProvider.kt
package com.tebet.mojual.view.history

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class HistoryFragmentProvider {

    @ContributesAndroidInjector(modules = [HistoryFragmentModule::class])
    internal abstract fun provideHistoryFragmentFactory(): HistoryFragment
}
